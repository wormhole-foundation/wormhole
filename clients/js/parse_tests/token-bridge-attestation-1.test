010000000001006cd3cdd701bbd878eb403f6505b5b797544eb9c486dadf79f0c445e9b8fa5cd474de1683e3a80f7e22dbfacd53b0ddc7b040ff6f974aafe7a6571c9355b8129b00000000007ce2ea3f000195f83a27e90c622a98c037353f271fd8f5f57b4dc18ebf5ff75a934724bd0491a43a1c0020f88a3e2002000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc200021200000000000000000000000000000000000000000000000000000000574554480000000000000000000000000000000000000057726170706564206574686572
