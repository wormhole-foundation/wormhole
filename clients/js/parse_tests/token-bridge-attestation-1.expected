{
  version: 1,
  guardianSetIndex: 0,
  signatures: [
    {
      guardianSetIndex: 0,
      signature: '6cd3cdd701bbd878eb403f6505b5b797544eb9c486dadf79f0c445e9b8fa5cd474de1683e3a80f7e22dbfacd53b0ddc7b040ff6f974aafe7a6571c9355b8129b00'
    }
  ],
  timestamp: 0,
  nonce: 2095245887,
  emitterChain: 1,
  emitterAddress: '0x95f83a27e90c622a98c037353f271fd8f5f57b4dc18ebf5ff75a934724bd0491',
  sequence: 11833801757748136510n,
  consistencyLevel: 32,
  payload: {
    module: 'TokenBridge',
    chain: 0,
    type: 'AttestMeta',
    tokenAddress: '0x000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
    tokenChain: 2,
    decimals: 18,
    symbol: 'WETH',
    name: 'Wrapped ether'
  },
  digest: '0x4bb52b9a44ff6062ba5db1c47afc40c186f7485c8972b1c6261eb070ce0b1c6e'
}
